content/services/index.rst

```diff
@@ -0,0 +1,111 @@
+public: yes
+location: development
+grid_template: true
+headline:
+  - tagline: 'Fully-integrated to achieve your goals'
+    type: 'Services'
+summary: |
+  **OddBird is a full-stack design & development agency**.
+  Our team is fully-integrated,
+  and ready to provide value from
+  the first day of a new project.
+  We'll help take you from concept to launch
+  with agile project management,
+  branding and content strategy,
+  interface and user-experience design,
+  and well-architected code.
+
+  .. callmacro:: content.macros.j2#link_button
+    :url: '/contact/'
+    :class: 'section-end'
+
+    Get started with a free consultation
+
+
+Application Design & Development
+================================
+
+.. callmacro:: content.macros.j2#divider
+
+.. callmacro:: content.macros.j2#get_quotes
+  :page: 'work/coachhub'
+  :slug: 'remote'
+
+.. callmacro:: content.macros.j2#divider
+
+.. callmacro:: content.macros.j2#rst
+  :tag: 'start'
+
+Shared Understanding of Goals & Constraints
```

Comment:
Instead of "Constraints" the word "Budget" may communicate better and address a big concern for lots of prospective clients.

Reply:
But I don't just mean budget… :/